templates/config/prod.exs

```diff
@@ -0,0 +1,6 @@
+import Config
+
+config :still,
+  dev_layout: false,
+  url_fingerprinting: true,
+  base_url: "YOUR_URL"
```

Comment:
@frm I'm not sure about how to handle this.

Reply:
Let's print a message with the generator.

Reply:
I really don't know what you're talking about. Can you help?